### PR TITLE
For #12007: Use 48dp height for search engine radio buttons

### PR DESCRIPTION
**Pull request by hakkikaancaliskan**
https://github.com/mozilla-mobile/fenix/issues/12007#issuecomment-653448867

resim

### Pull Request checklist

- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### After merge
- [x] **Milestone**: Make sure issues finished by this pull request are added to the [milestone](https://github.com/mozilla-mobile/fenix/milestones) of the version currently in development.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

File: app/src/main/java/org/mozilla/fenix/settings/search/AddSearchEngineFragment.kt [hunk 1]
```diff
@@ -93,7 +93,7 @@ class AddSearchEngineFragment : Fragment(), CompoundButton.OnCheckedChangeListen
 
         availableEngines.forEachIndexed(setupSearchEngineItem)
 
-        val engineItem = makeCustomButton(layoutInflater)
+        val engineItem = makeCustomButton(layoutInflater, res = requireContext().resources)
```
Comment (NotWoods) [inline app/src/main/java/org/mozilla/fenix/settings/search/AddSearchEngineFragment.kt:96]:
```suggestion
        val engineItem = makeCustomButton(layoutInflater, res = resources)
```

File: app/src/main/java/org/mozilla/fenix/settings/search/AddSearchEngineFragment.kt [hunk 2]
```diff
@@ -250,11 +250,12 @@ class AddSearchEngineFragment : Fragment(), CompoundButton.OnCheckedChangeListen
         toggleCustomForm(selectedIndex == -1)
     }
 
-    private fun makeCustomButton(layoutInflater: LayoutInflater): View {
+    private fun makeCustomButton(layoutInflater: LayoutInflater, res: Resources): View {
         val wrapper = layoutInflater
             .inflate(R.layout.custom_search_engine_radio_button, null) as ConstraintLayout
         wrapper.setOnClickListener { wrapper.radio_button.isChecked = true }
         wrapper.radio_button.setOnCheckedChangeListener(this)
+        wrapper.minHeight = res.getDimension(R.dimen.radio_button_preference_height).toInt()
```
Comment (NotWoods) [inline app/src/main/java/org/mozilla/fenix/settings/search/AddSearchEngineFragment.kt:258]:
```suggestion
        wrapper.minHeight = res.getDimensionPixelSize(R.dimen.radio_button_preference_height)
```

**Comment by hakkikaancaliskan** [comment]
@NotWoods I'm on a travel, i don't have my laptop. Could someone merge this with fixing nits?

File: app/src/main/java/org/mozilla/fenix/settings/search/AddSearchEngineFragment.kt [hunk 3]
```diff
@@ -280,6 +281,7 @@ class AddSearchEngineFragment : Fragment(), CompoundButton.OnCheckedChangeListen
         engineIcon.setBounds(0, 0, iconSize, iconSize)
         wrapper.engine_icon.setImageDrawable(engineIcon)
         wrapper.overflow_menu.visibility = View.GONE
+        wrapper.minHeight = res.getDimension(R.dimen.radio_button_preference_height).toInt()
```
Comment (NotWoods) [inline app/src/main/java/org/mozilla/fenix/settings/search/AddSearchEngineFragment.kt:284]:
```suggestion
        wrapper.minHeight = res.getDimensionPixelSize(R.dimen.radio_button_preference_height)
```

**Comment by ekager** [comment]
Closing for https://github.com/mozilla-mobile/fenix/pull/12323

**Pull request closed without merging**